frontend/src/components/Tabcordian.tsx

```diff
@@ -1,107 +1,118 @@
-import Link from "next/link"
-import { FC, useState } from "react"
+import { FC, useState } from 'react'
+
+import Link from 'next/link'
 import { MdKeyboardArrowDown, MdKeyboardArrowRight } from 'react-icons/md'
 
 interface TabcordianLink {
-    title: string
-    url: string
-    description: string
+  title: string
+  url: string
+  description: string
 }
 
 interface TabcordianData {
-    title: string
-    description: Array<string>
-    linksTitle: string
-    links: Array<TabcordianLink>
+  title: string
+  description: Array<string>
+  linksTitle: string
+  links: Array<TabcordianLink>
 }
 
 export interface TabcordianProps {
-    data: Array<TabcordianData>
+  data: Array<TabcordianData>
 }
 
-const Tabcordian: FC<TabcordianProps> = ({
-    data,
-  }: TabcordianProps) => {
+const Tabcordian: FC<TabcordianProps> = ({ data }: TabcordianProps) => {
+  const [active, setActive] = useState<number | null>(null)
```

Comment:
Correct me if I'm wrong, but if the user refreshes the page, the active tab will be lost. Consider updating the URL by adding something like `#tab={tabid}` and use that so that if the user refreshes, the tab remains active.

Reply:
I don't see this as a bad UX. We could add extra logic to handle this but is it worth it? You also need to not update the browser history on tab change, because the user could change tabs like 20 times and trying to going the previous page but would take like 20 times because of the browser history. Anyway, this is not purpose of this PR.

Reply:
Okay. I'll resolve the comment since this isn't the purpose of your PR.